cartoframes/magic.py

```diff
@@ -0,0 +1,82 @@
+"""carto magics"""
+from IPython.core.magic import (Magics, magics_class, line_magic,
+                                cell_magic, line_cell_magic)
+from cartoframes.context import CartoContext
+from cartoframes import utils
+# GOALS
+# - have it automatically find a CartoContext object
+# - make it agnostic to line or cell magics
+# - maps!
+
+def null_to_none(string):
```

Comment:
is this needed?

Reply:
This is for the first checkbox, where you suggested we have a  small function in the magics.py module that's similar to utils.pgquote but does None instead of 'null'.

Reply:
Yeah, but I don't see it being used anywhere below

Reply:
True! I just inserted an`if` statement to trigger the `utils.pgquote` function in the case that `stylecol` and `timecol` were `None`. I will remove this.